wallet-core/contributing.md

```diff
@@ -135,22 +135,24 @@ The wallet core code generator also parses coin configuration defined in `coins.
 
 ```json
 {
-    "id": "bitcoin",                            // coin id
-    "name": "Bitcoin",                          // coin name
-    "symbol": "BTC",                            // coin symbol
-    "decimals": 8,                              // minimum currency unit, e.g. a satoshi is 0.00000001 ฿
-    "blockchain": "Bitcoin",                    // blockchain type
-    "derivationPath": "m/84'/0'/0'/0/0",        // default path for derivating private key from a mnemonic
```

Comment:
@hewigovens I would add a mention about BIP44/SLIP44 in here, to make the developer aware of it

Reply:
yes, we will add a `slip44` field to fix https://github.com/TrustWallet/wallet-core/issues/486